test-clients/src/main/resource/umbrellaTest.properties

```diff
@@ -0,0 +1,121 @@
+# port of node; default value is 50211
```

Comment:
Is this properties file used anywhere?

Reply:
Yes Leo, Used for Legacy smartContact tests. I Had to move it here as I need all the required files to run from suite runner in `src/main/resource` folder because this folder is copied to all test clients when using JRS.

Reply:
I see.